README.md

```diff
@@ -1,51 +1,11 @@
+# Wrangler Monorepo Documentation
+
 ## 🤠 wrangler
 
-`wrangler` is a command line tool for building [Cloudflare Workers](https://workers.cloudflare.com/).
+`wrangler` is a command line tool for building [Wrangler Documentation](./packages/wrangler/README.md).
```

Comment:
?? wrangler isn't a tool for making documentation. huh?

Reply:
This doc needs to be fleshed out before it lands, it's very sparse right now and doesn't say anything.

Reply:
True, wasn't sure what to put for the monorepo documentation.

Reply:
> ?? wrangler isn't a tool for making documentation. huh?

This was an error. Updated it.